src/index.js

```diff
@@ -103,68 +109,62 @@ class Game extends React.Component {
   }
   
   render() {
-	const history = this.state.history;
-  const current = history[this.state.stepNumber];
-  const winner = calculateWinner(current.squares);
-  let Xcoloring = 1;
-	
-	const moves = history.map((step, move) => {
-      const desc = move ?
-        'Go to move #' + move + ' (' + (move % 2 ? 'X' : 'O') + ' at position ' + history[move].location + ')':
-        'Go to game start';
-      if (move % 2) {
-        return (
-          <li key={move}>
-            <button onClick={() => this.jumpTo(move)}>
-            {move === this.state.stepNumber ? <b className = "highlightedX" style = {{marginLeft:"0"}}>{desc}</b> : desc}
-            </button>
-          </li>
-        );
-      } else {
+    const boardSize = this.state.boardSize;
+    const history = this.state.history;
+    const current = history[this.state.stepNumber];
+    const winner = calculateWinner(current.squares, boardSize);
```

Comment:
I would change this to have.

`getGameStatus(...)` - playing || draw || end (or win/lost if looking from players perspective)
`getWinner(...)` - null || winner

You always want to be as explicit as you can. Function should do one thing, variable should communicate one thing. If you use winner as the game status then you never have status by itself.

Reply:
Fixed the variable and function names to be more explicit and understandable, appreciate it